### DISCUSSION TITLE
remote: QUIT error

### DISCUSSION THREAD

**Discussion by cbrake** [emersion/maddy#476] [Q&A] [answered]
I see this when sending mail to outlook users. The mail is delivered, so seems to functionally work, but curious what the QUIT error is about:

```
Apr 01 20:15:29 mail5 maddy[122451]: remote: QUIT error        {"msg_id":"012dc539-62475050","reason":"write tcp 173.230.132.137:51912-\u003e104.47.59.138:25: write: broken pipe","remote_server":"somecompany.mail.protection.outlook.com."}
```

**Comment by foxcpp** [comment, marked as answer]
It is maddy trying to close idle SMTP connections "properly" while the remote server already closed the connection. This happens because maddy keeps a connection open after sending a message to reuse it if another message needs to be sent to the same domain, but SMTP servers frequently close idle connections before maddy checks whether it is still open.

TLDR Nothing to worry about, unless you send messages to 10k different domains daily (in which case the amount of half-open connections left by maddy might be an issue).

**Reply by cbrake** [reply]
thanks!
